resolvers/deno/lockfile.rs

```diff
@@ -282,6 +282,9 @@ impl<TSys: LockfileSys> LockfileLock<TSys> {
           PackageJsonDepValue::Req(req) => {
             Some(JsrDepPackageReq::npm(req.clone()))
           }
+          PackageJsonDepValue::JsrReq(req) => {
+            Some(JsrDepPackageReq::jsr(req.clone()))
```

Comment:
Maybe let's have this error for the time being? I think it needs to actually be an npm package depending on the `@jsr` scope.

Reply:
Yeah I just ended up doing the warning